src/components/MultiSelect/MultiSelect.scss

```diff
@@ -1,10 +1,80 @@
 @use "sass:map";
-@import "vanilla-framework";
-@include vf-base;
-@include vf-p-lists;
+
+@import "~vanilla-framework/scss/_base_forms";
+@import "~vanilla-framework/scss/base_placeholders";
+@import "~vanilla-framework/scss/base_icon-definitions";
+
+@include vf-b-placeholders;
 
 $dropdown-max-height: 20rem;
 
+/**
+ Including base vanilla input elements declaration from:
+https://github.com/canonical/vanilla-framework/blob/b4378c920f8bcb804db2ead64566ce3fa54636ae/scss/_base_forms.scss#L14
+
+Workaround for: https: //github.com/canonical/react-components/issues/1041
+*/
+
+%vf-readonly-element {
```

Comment:
Why do we need to copy so much from Vanilla here?
This is extremely risky and fragile, as these internals of Vanilla are likely to be changed and this copy will not reflect that.

And I'm not sure if I understand how these styles are related to completely separate Slider component being broken?

Reply:
I'm fully aware it's very fragile, hence I clearly marked it as a workaround in the comment. This was meant to be a quick fix until we figure out a proper way of addressing this. I'll add more information to the linked issue.